CGATPipelines/pipeline_peakcalling/pipeline.ini

```diff
@@ -33,10 +33,10 @@ input=1
 #
 ################################################################
 [annotations]
-database=!?
+database=/ifs/mirror/annotations/hg19_ensembl75/csvdb
```

Comment:
According to what we are discussing in other pull requests, don't we need to leave this empty for portability?